src/main/app/fees/feesClient.ts

```diff
@@ -22,9 +22,8 @@ export class FeesClient {
    * @param {number} claimValue the amount claiming for in pounds
    * @returns {Promise.<number>} promise containing the fee amount in pounds
    */
-  static calculateIssueFee (claimValue: number): Promise<number> {
-    return this.calculateFee(issueFeeCode, claimValue)
-      .then((outcome: CalculationOutcome) => MoneyConverter.convertPenniesToPounds(outcome.amount))
+  static async calculateIssueFee (claimValue: number): Promise<number> {
+    return MoneyConverter.convertPenniesToPounds(((await this.calculateFee(issueFeeCode, claimValue))).amount)
```

Comment:
can you extract to a variable the fee calculation this is quite hard to read?
REC

Reply:
I agree

Reply:
done